src/Plugin.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * The main plugin class.
+ */
+
+namespace Geniem\Oopi;
+
```

Comment:
... to this class so everything works correctly 👍

Reply:
Sorry. This is the first time I use a computer 🙃